action/scenario_sequence/src/event.cpp

```diff
@@ -32,7 +32,7 @@ Event::Event(
 simulation_is Event::update(
   const std::shared_ptr<scenario_intersection::IntersectionManager>&)
 {
-  if (ignited_ = condition_.evaluate(context_))
+  if ((ignited_ = condition_.evaluate(context_)))
```

Comment:
out of curiosity: was there a warning that parentheses should be added?

Reply:
Yes. I hope the intention is indeed to do an assignment.

Reply:
that's not clear. The only idiomatic use for assignment inside `if` clause that I can think of is when a pointer is returned and it may be  `nullptr`. 
In this case, it looks like it should be an assignment, `yamacir-kit` would know as the author of this line. Let's leave it as you have it